app/src/org/commcare/preferences/GlobalPrivilegesManager.java

```diff
@@ -76,4 +94,42 @@ private static String getPrivilegeDisplayName(String privilegeName) {
         }
     }
 
+    public static void storeRetrievedAvailableApps(Vector<AppAvailableForInstall> availableAppsRetrieved) {
```

Comment:
Seems like these don't necessarily fit in with the `GlobalPriviledgesManager` class (and ideally would be stored in a SQLite DB rather than the SharedPreferences?) - but seems fine for a temporary, low-touch home